### DISCUSSION TITLE
v4 is not published to npm?

### DISCUSSION THREAD

**Discussion by jbinto** [bergie/passport-saml#807] [Q&A] [answered]
Just wondering if this is intentional or not. It doesn't seem like any v4 release is published to npm, latest I can see is 3.2.4.

**Comment by srd90** [comment, marked as answer]
@jbinto it is published as [@node-saml/passport-saml](https://www.npmjs.com/package/@node-saml/passport-saml)

See also #729

**Reply by cjbarth** [reply]
passport-saml <= 4 has been deprecated: https://www.npmjs.com/package/passport-saml
